pkg/webhook/indexeres/indexer.go

```diff
@@ -114,3 +119,11 @@ func scheduleVMBackupByCronGranularity(obj *harvesterv1.ScheduleVMBackup) ([]str
 
 	return []string{granularity.String()}, nil
 }
+
+func imageByStorageClass(obj *harvesterv1.VirtualMachineImage) ([]string, error) {
+	sc, ok := obj.Annotations[util.AnnotationStorageClassName]
```

Comment:
I'm not sure if the Annotations can be nil, if yes, I suggest checking if the Annotations is nil.

Reply:
We don't need to.

In Go, we can read a nil map, but can't write any value into nil map. 

reading nil map https://go.dev/play/p/AhYjSZ7DpQi
writing nil map https://go.dev/play/p/7TSdpOwxCvT

Reply:
Sounds great, thanks for the check.